scripts/hudson/narayana.sh

```diff
@@ -124,13 +124,17 @@ function build_narayana {
   echo "Building Narayana"
   cd $WORKSPACE
 
+  # JAVA_HOME_FOR_BUILD can be used in environment if JDK for building Narayana needs to be different from one which runs e.g. tests.
+  # This env property is used e.g. to build Narayana with JDK 1.7 but run the tests with JDK 1.6.
+  # Narayana targets 1.6 byte code. And the java source is limited to 1.6 language features only.
+  # We must build on 1.7 because we are using a JDK API that only exists in JDK 1.7.
   if [ -n "$JAVA_HOME_FOR_BUILD" ]; then
     local pathOriginal="$PATH"
     local javaHomeOriginal="$JAVA_HOME"
     export JAVA_HOME="$JAVA_HOME_FOR_BUILD"
     export PATH="$JAVA_HOME/bin:$PATH"
   fi
-
+FailAfterPrepareBase.java
```

Comment:
This looks wrong to me, is it expected?

Reply:
+1, thanks for spotting this. This is a dumb copy&paste error.

Reply:
https://github.com/jbosstm/narayana/pull/1733